helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/BestPossibleExternalViewVerifier.java

```diff
@@ -224,12 +224,15 @@ public boolean verifyByZkCallback(long timeout) {
   @Override
   protected synchronized boolean verifyState() {
     try {
+      LOG.debug("Verifier start verifyState at {}", System.currentTimeMillis());
```

Comment:
Do you need System.currentTimeMillis()?  Will the log4j print out the timestamp?

Reply:
good point. Removed time stamp for all of them.

Reply:
The timestamp seems not to be removed.

And I really don't like these outputs even as debug. I think they are only useful when debugging. And we shall not add them to the repo.